packages/core/src/modules/question-answer/QuestionAnswerModule.ts

```diff
@@ -97,6 +102,15 @@ export class QuestionAnswerModule {
     dispatcher.registerHandler(new AnswerMessageHandler(this.questionAnswerService))
   }
 
+  private registerFeatures(featureRegistry: FeatureRegistry) {
```

Comment:
More a note for @TimoGlastra but we should probably add this method to the `BaseModule` or w/e it is called. Makes implementing a custom module with feature discovery a bit easier.

Reply:
Yes! I did some tries adding a static method for feature registration (similar to `register(dependencyManager)`) but was not convinced yet.

Reply:
[Here](https://github.com/hyperledger/aries-framework-javascript/issues/495#issuecomment-1220126118) some discussion about it.

Reply:
Yes agreed. However we should probably only add it in 0.3.0 with the custom modules